internal/runners/config/set.go

```diff
@@ -38,9 +38,17 @@ func (s *Set) Run(params SetParams) error {
 	}
 	switch option.Type {
 	case configMediator.Bool:
-		value = cast.ToBool(params.Value)
+		var err error
+		value, err = strconv.ParseBool(params.Value)
```

Comment:
The `cast` library also has a `ToBoolE` function but since params.Value is not an interface I'm not sure we would get any benefit from using it.